.eslintrc.json

```diff
@@ -101,6 +99,47 @@
           }
         ]
       }
+    },
+    {
+      "files": [
+        "**/*.{md,mdx}"
+      ],
+      "extends": [
+        "plugin:mdx/recommended"
+      ],
+      "settings": {
+        "mdx/code-blocks": true
+      }
+    },
+    {
+      "files": [
+        "**/*.{md,mdx}",
+        "**/*.{md,mdx}/**"
```

Comment:
Why are we using two separate override objects?

Reply:
The code blocks in markdown (and mdx) are split into separate (virtual) files for eslint.

/docs/file.md with one python, one jsx and one TS converts to something like:

/docs/file.md/1.py
/docs/file.md/2.jsx
/docs/file.md/3.ts

But actually here it would only be necessary to include the second as these rules shouldn't be applied to the markdown itself, only to the matching javascript code snippets. Will clean it up.